stdlib/asyncio/protocols.pyi

```diff
@@ -1,28 +1,29 @@
 import sys
 from asyncio import transports
+from typing import Protocol as TypingProtocol
```

Comment:
Why should asyncio's protocols be typing protocols? Does that solve a concrete problem? Is it common to define classes that have compatible methods but don't inherit from asnycio's protocol classes?

Even if this is a good idea, I would prefer this to be a separate pull request, as it doesn't have anything to do with making classes abstract.

Reply:
You're right, I'll move the changes to this file into a new PR, with an explanation as to why the changes are being made.

I originally made the methods on these protocols abstract too, but it wasn't a good approach; so those changes no longer belong in this PR.